cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -52,6 +52,18 @@ bool gdv_fn_like_utf8_utf8_utf8(int64_t ptr, const char* data, int data_len,
   return (*holder)(std::string(data, data_len));
 }
 
+const char* gdv_fn_regexp_extract_utf8_int32(int64_t ptr, int64_t holder_ptr,
```

Comment:
Same here:
   Rename the method to "gdv_fn_regexp_extract_utf8_utf8_int32"

Reply:
The changes were applied